src/_includes/components/sub-banner.njk

```diff
@@ -0,0 +1,23 @@
+<ul>
```

Comment:
Tiny thing, but can we change the name of this file to something that's more specific? This is the cohort pages sub navigation, so how about something like `cohort-subnav.njk`

Reply:
Changed the name!